benchmarks/nel/scripts/datasets/mewsli_9.py

```diff
@@ -68,12 +68,26 @@ def _create_annotated_docs(self) -> List[Doc]:
                         self._paths["assets"] / "en" / "text" / row["docid"],
                         encoding="utf-8",
                     ) as text_file:
+                        lines = text_file.readlines()
+                        doc_text = " ".join(
+                            " ".join(
+                                [
+                                    line.replace("\n", " ").replace("\xa0", ".")
+                                    for line in lines
+                                ]
+                            ).split()
+                        )
```

Comment:
What is this code supposed to do? Also is it correct to replace `\xa0` (non-breaking space) with a period and not a space?

Reply:
`\xa0` seems to be only used after a title in the Mewsli-9/Wikinews corpus, and there usually isn't any end-of-sentence punctuation after the title. I thought that converting `\xa0` into a a period instead might help the pipeline with parsing the text correctly.

This snippet, after replacing special characters with spaces/periods, condenses multiple to single whitespaces by splitting and then joining the document text.